### PR TITLE
chore: adding tags to test at Feature level

### PR DESCRIPTION
**Pull request by atharvar28**
Adding Tags to individual tests at Feature level.

What happens here? - so if the service like fence is not deployed to the commons, any test related to fence shouldnt be executed. This tagging will allow that to happen.

### CODE REVIEW

File: suites/apis/metadataIngestionTest.js [hunk 1]
```diff
@@ -6,7 +6,7 @@
     within the sower config block (ingest-metadata-manifest & get-dbgap-metadata)
  3. Metadata Service (mds) must also be deployed
 */
-Feature('Metadata Ingestion');
+Feature('Metadata Ingestion @requires-sower');
```
Comment (haraprasadj) [inline suites/apis/metadataIngestionTest.js:9]: does this also require mds / indexd / fence?

File: suites/apis/exportToWorkspaceTest.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Feature('ExportToWorkspaceAPITest').retry(2);
+Feature('ExportToWorkspaceAPITest @requires-fence').retry(2);
```
Comment (haraprasadj) [inline suites/apis/exportToWorkspaceTest.js:1]:
```suggestion
Feature('ExportToWorkspaceAPITest @requires-fence @requires-manifestservice').retry(2);
```

File: suites/apis/dbgapTest.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*eslint-disable */
-Feature('dbgapSyncing').retry(2);
+Feature('dbgapSyncing @requires-fence').retry(2);
```
Comment (haraprasadj) [inline suites/apis/dbgapTest.js:2]:
```suggestion
Feature('dbgapSyncing @requires-fence @requires-indexd').retry(2);
```

File: suites/apis/centralizedAuth.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /*eslint-disable */
-Feature('CentralizedAuth');
+Feature('CentralizedAuth @requires-fence');
```
Comment (haraprasadj) [inline suites/apis/centralizedAuth.js:2]: does this require arborist too?

File: suites/apis/metadataIngestionTest.js (hunk 1 above)
Comment (atharvar28) [reply suites/apis/metadataIngestionTest.js:9]: This is an initial WIP PR. We can keep adding stuff to this

File: suites/apis/centralizedAuth.js (hunk 4 above)
Comment (atharvar28) [reply suites/apis/centralizedAuth.js:2]: It also needs `Indexd`

**Review by jingh8 — DISMISSED** [review]
+1

**Review by haraprasadj — APPROVED** [review]

**Pull request merged**